chart/values.yaml

```diff
@@ -436,3 +436,7 @@ eirinix:
     icon_image: Eirini broker
     secrets:
       auth-password: ~ # Password is randomly generated if not given
+  dns-aliases:
```

Comment:
Can you include these details in here?  https://github.com/cloudfoundry-incubator/kubecf/blob/master/mixins/eirinix/config/eirinix.yaml

Like all the other eirinix components.